src/Shopsys/ShopBundle/Resources/translations/validators.de.po

```diff
@@ -2,155 +2,151 @@ msgid ""
 msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Language: en\n"
+"Language: de\n"
 
 msgid "City name cannot be longer than {{ limit }} characters"
-msgstr "(de) City name cannot be longer than {{ limit }} characters"
+msgstr "Der Name der Stadt darf nicht länger als {{ limit }}} Zeichen sein"
```

Comment:
Twig variables have to be in two (2) curly brackets. This rule is often violated.